account_move_exception/data/account_exception_data.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <!-- Test Purchase Exceptions Scheduler-->
+    <record model="ir.cron" forcecreate="True" id="ir_cron_test_acc_move_except">
+        <field name="name">Account: Test Draft Moves Exception</field>
+        <field name="model_id" ref="account.model_account_move" />
+        <field name="state">code</field>
+        <field name="code">model.test_all_draft_moves()</field>
+        <field name="user_id" ref="base.user_root" />
+        <field name="interval_number">20</field>
+        <field name="interval_type">minutes</field>
+        <field name="numbercall">-1</field>
+        <field name="doall" eval="False" />
+        <field name="active" eval="False" />
+    </record>
+    <record id="am_excep_no_email" model="exception.rule">
+        <field name="name">No email on vendor</field>
+        <field name="description">No Email for Vendor</field>
+        <field name="sequence">50</field>
+        <field name="model">account.move</field>
+        <field name="code">if not self.partner_id.email:
+            failed=True</field>
+        <field name="active" eval="False" />
+    </record>
+    <record id="aml_excep_qty_check" model="exception.rule">
+        <field name="name">Quantity not negative</field>
```

Comment:
Change name to Quantity not positive or Quantity negative

Reply:
Done!